mithril-test-lab/mithril-end-to-end/src/mithril/signer.rs

```diff
@@ -55,6 +55,7 @@ impl Signer {
                 signer_config.pool_node.db_path.to_str().unwrap(),
             ),
             ("DATA_STORES_DIRECTORY", &data_stores_path),
+            ("STORE_RETENTION_LIMIT", "10"),
```

Comment:
Maybe we should use a bigger retention to facilitate debug investigations? 
```suggestion
            ("STORE_RETENTION_LIMIT", "50"),
```

Reply:
The goal is to find a balance between doing no pruning at all in the e2e and not doing them too much. 10 seems to strike that balance since e2e usually start producing data about epoch 6~7 and finish around epoch 20~22.

Reply:
OK, maybe `15` would be a better value in that case. 
But as we can change this value easily if needed, let's keep `10` for now :+1: